react/SlideToggle/SlideToggle.less

```diff
@@ -14,43 +18,45 @@
 }
 
 .input {
+  opacity: 0;
   position: absolute;
-  left: 10px;
-  top: 5px;
+  left: 1px;
   margin: 0;
+  width: @toggleWidth;
+  height: @toggleHeight;
 
-  &:focus ~ .slider .slideButton::before {
+  &:focus ~ label .slider .slideButton::before {
```

Comment:
If we [do this](https://github.com/seek-oss/seek-style-guide/pull/449/files#r176975817) we can move away from using the global element selectors